migrations/V007__add_image_key_columns.sql

```diff
@@ -0,0 +1,8 @@
+ALTER table league
+ADD COLUMN image_key VARCHAR(255) NOT NULL AFTER technical_name,
+ADD COLUMN medium_image_key VARCHAR(255) NOT NULL AFTER image_key,
+ADD COLUMN small_image_key VARCHAR(255) NOT NULL AFTER medium_image_key;
```

Comment:
So these are base urls? The name makes me think they’re gonna be some sort of identifier

Reply:
I had the same thought. Keys are usually either technical ids or translation keys

Reply:
oh yeah, I should probably rename them to urls